src/server/saved_objects/service/lib/repository.test.js

```diff
@@ -1664,12 +1696,154 @@ describe('SavedObjectsRepository', () => {
       onBeforeWrite.throws(es401);
 
       try {
-        await savedObjectsRepository.delete('type', 'id');
+        await savedObjectsRepository.delete('foo', 'id');
       } catch (error) {
         sinon.assert.calledOnce(onBeforeWrite);
         expect(error).toBe(es401);
         expect(errors.isNotAuthorizedError(error)).toBe(true);
       }
     });
   });
+
+  describe('unsupported types', () => {
+    it('should error when attempting to \'get\' an unsupported type', async () => {
+      await expect(savedObjectsRepository.get('hiddenType')).rejects.toEqual(new Error('Not Found'));
+    });
+
+    it('should error when attempting to \'bulkGet\' an unsupported type', async () => {
```

Comment:
The test description does not match the scenario. The test expects a valid response, whereas the description expects an error

Reply:
I knew I had missed at least one of these. Thanks!